.github/workflows/deploy.yml

```
@@ -0,0 +1,139 @@
+        with:
+          credentials_json: ${{ secrets.GCP_SA_KEY }}
+
+      - name: Set up Cloud SDK
+        uses: google-github-actions/setup-gcloud@v2
+
+      - name: Configure Docker authentication
+        run: |-
+          gcloud auth configure-docker ${{ env.REGION }}-docker.pkg.dev --quiet
+
+      - name: Set Image Tag
+        run: echo "IMAGE_TAG=${{ env.REGION }}-docker.pkg.dev/${{ env.PROJECT_ID }}/${{ env.REPO_NAME }}/${{ env.SERVICE_NAME }}:${GITHUB_SHA::7}" >> $GITHUB_ENV
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:1:138: Double quote to prevent globbing and word splitting

(shellcheck)